CONTRIBUTING.md

```diff
@@ -41,6 +41,16 @@ return it. Once we receive it, we'll be able to accept your pull requests.
 
 ***NOTE***: Only original source code from you and other people that have signed the CLA can be accepted into the main repository.
 
+### Setting up an environment
+1. Install [pyenv](https://github.com/pyenv/pyenv#installation)
+2. Install python 3.9 `pyenv install 3.9`
+3. Locate the python installation ex: `~/.pyenv/versions/3.9.9/bin/python3.9`
+4. Create a virtual environment `<PATH_TO_PYTHON> -m venv venv`
```

Comment:
Perhaps steps 3/4 could be improved with `pyenv global 3.9.9` (or whatever version from previous step), and then just plain `python -mvenv venv` rather than needing to use the `~/.pyenv` path.

(I'm not so familiar with `pyenv` and how it interacts with `venv`, but perhaps `pyenv local 3.9.9` in the Arelle checkout is cleaner than `pyenv global`.  If so, we could perhaps add `.pyenv-version` to `.gitignore` so the marker doesn't accidentally get checked in.)

Reply:
Could also use `PYENV_VERSION=3.9.9 pyenv exec python -m venv venv`.

Reply:
Yes, I think that's a great suggestion, thanks.